components/static-pages/privacypolicy.md

```diff
@@ -1,11 +1,15 @@
 # Open Collective Privacy Policy
 
-**Last Updated: 7 January 2019**
+**Last Updated: 2025-02-26**
 
-<p class="lead">This Privacy Policy explains how information about you is collected, used, and disclosed by Open Collective, Inc. ("Open Collective" or “we”). This Privacy Policy applies to information we collect when you use our websites and online services (collectively, the “Services”) or when you otherwise interact with us.</p>
+<p class="lead">This Privacy Policy explains how information about you is collected, used, and disclosed by OFi Technologies LLC ("OFi Technologies", "we", "us", or "our"), a company 100% owned and controlled by Open Finance Consortium, a C3 non-profit. The opencollective.com website is operated by OFi Technologies LLC. This Privacy Policy applies to information we collect when you use our websites and online services (collectively, the "Services") or when you otherwise interact with us.</p>
```

Comment:
Policy then goes on to talk about Open Collective but Open Collective is not defined.